### PR TITLE
dataclient/kubernetes: reduce amount of logs

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Log Ingress/RouteGroup processing messages at most once per minute unless log level is debug.

### CODE REVIEW

File: dataclients/kubernetes/kube.go [hunk 1]
```diff
@@ -391,16 +392,20 @@ func (c *Client) loadAndConvert() ([]*eskip.Route, error) {
 		return nil, err
 	}
 	c.state = state
+	previousUpdate := c.lastUpdate
+	c.lastUpdate = time.Now()
 	c.mu.Unlock()
 
+	enableLogging := log.GetLevel() >= log.DebugLevel || time.Since(previousUpdate) > 1*time.Minute
```
Comment (AlexanderYastrebov) [inline dataclients/kubernetes/kube.go:399]: @AlexanderYastrebov where is the test for this :see_no_evil:

Comment (szuecs) [reply dataclients/kubernetes/kube.go:399]: I don't see it :)

**Comment by szuecs** [comment]
Interesting pipeline error...

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
> Interesting pipeline error...

yep
```
--- FAIL: TestTeeEndToEndBody2TeeRoutesAndClosing (0.00s)
    tee_test.go:265: shadow handler got a request, but should not
panic: close of closed channel [recovered]
	panic: close of closed channel

goroutine 50 [running]:
testing.tRunner.func1.2({0x128aa60, 0x14e1d80})
	/usr/local/go/src/testing/testing.go:1526 +0x24e
testing.tRunner.func1()
	/usr/local/go/src/testing/testing.go:1529 +0x39f
panic({0x128aa60, 0x14e1d80})
	/usr/local/go/src/runtime/panic.go:884 +0x213
runtime.Goexit()
	/usr/local/go/src/runtime/panic.go:522 +0x177
testing.(*common).FailNow(0xc00028cb60)
	/usr/local/go/src/testing/testing.go:980 +0x4c
testing.(*common).Fatal(0xc00028cb60, {0xc00014db88?, 0x13afa38?, 0xc0002927f0?})
	/usr/local/go/src/testing/testing.go:1057 +0x58
github.com/zalando/skipper/filters/tee.TestTeeEndToEndBody2TeeRoutesAndClosing(0xc00028cb60)
	/workspace/filters/tee/tee_test.go:265 +0xe8f
testing.tRunner(0xc00028cb60, 0x140b158)
	/usr/local/go/src/testing/testing.go:1576 +0x10b
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1629 +0x3ea
FAIL	github.com/zalando/skipper/filters/tee	0.014s
FAIL
/bin/sh: 1: break: Illegal number: -1
```

File: dataclients/kubernetes/kube.go (hunk 1 above)
Comment (AlexanderYastrebov) [reply dataclients/kubernetes/kube.go]: Added a proper test for it.

**Comment by lucastt** [comment]
👍

**Comment by szuecs** [comment]
:+1:

File: dataclients/kubernetes/kube.go (hunk 1 above)
Comment (AlexanderYastrebov) [reply dataclients/kubernetes/kube.go]: See https://github.com/zalando/skipper/pull/2427 :facepalm:

**Pull request merged**